test/plots/piePlotTests.ts

```diff
@@ -435,8 +401,6 @@ describe("Plots", () => {
           { x: 230, y: 25 },
           { x: 245, y: 25 }
         ];
-        piePlot.removeDataset(simpleDataset);
-        piePlot.addDataset(new Plottable.Dataset(data));
         clicks.forEach((point: Plottable.Point, i: number) => {
           let entity = piePlot.entitiesAt(point);
           assert.strictEqual(entity.length, 1, "exactly one entity is selected");
```

Comment:
Same here. If 0 or >1 entities are selected, we cannot tell which, because there is no index in the assert message